tests/dualtor/test_orchagent_active_tor_downstream.py

```diff
@@ -73,7 +73,7 @@ def remove_neighbor(ptfhost, duthost, server_ip, ip_version):
             restore = True
             neighbor_advertise_process = "garp_service"
         elif ip_version == "ipv6":
-            restore = False
+            restore = True
```

Comment:
Is ipv6 neighbor statically configured similar with ipv4 neighbor?

Reply:
Yes, pls check: https://github.com/Azure/sonic-mgmt/blob/1193c9925ba928b991375f3ffabc68562edee5f6/tests/common/dualtor/dual_tor_mock.py#L280